src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1560,19 +1562,12 @@ def aks_browse(cmd, client, resource_group_name, name, disable_browser=False,
 
     logger.warning('Press CTRL+C to close the tunnel...')
     if not disable_browser:
-        wait_then_open_async(proxy_url)
+        wait_then_open_async(dashboardURL)
     try:
         try:
-            subprocess.check_output(["kubectl", "--kubeconfig", browse_path, "--namespace", "kube-system",
-                                     "port-forward", "--address", listen_address, dashboard_pod,
-                                     "{0}:{1}".format(listen_port, dashboard_port)], stderr=subprocess.STDOUT)
+            subprocess.check_output(["kubectl", "--kubeconfig", browse_path, "proxy"], stderr=subprocess.STDOUT)
         except subprocess.CalledProcessError as err:
-            if err.output.find(b'unknown flag: --address'):
-                if listen_address != '127.0.0.1':
-                    logger.warning('"--address" is only supported in kubectl v1.13 and later.')
-                    logger.warning('The "--listen-address" argument will be ignored.')
-                subprocess.call(["kubectl", "--kubeconfig", browse_path, "--namespace", "kube-system",
-                                 "port-forward", dashboard_pod, "{0}:{1}".format(listen_port, dashboard_port)])
```

Comment:
Why replace these lines with `logger.warning("error occured when setup the proxy: %s", err)`

Reply:
that's because we will not use the "--address" option now.
so the err.output will never have unknown flag: --address anymore.

Reply:
oh, wait, I need to add the -p option